shell/platform/windows/flutter_window.h

```diff
@@ -0,0 +1,120 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOW_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_FLUTTER_WINDOW_H_
+
+#pragma once
```

Comment:
Why is this needed? We use header guards for this, and the file already has them.

Reply:
Fixed in next update